src/main/java/codeu/model/data/ConversationStatBot.java

```diff
@@ -47,11 +86,41 @@ public String getMentionKey() {
   }
 
   /** Creates a reply based on the keyword. */
-  public String answerMessage(String message) {
+  public String answerMessage(String message, Conversation conversation) {
     String keyword = getKeyword(message);
 
     if (keyword == null) {
-      return "Stats loading...";
+      return "I'm sorry. I didn't understand that. Send a message with the word \"Help\".";
+    }
+
+    String botAnswer = answerMap.get(keyword);
+
+    if (keyword.equalsIgnoreCase("owner")) {
+      return String.format(botAnswer, getOwner(conversation.getOwnerId()));
+    }
+    else if (keyword.equalsIgnoreCase("created")) {
+      return String.format(botAnswer, getCreation(conversation.getCreationTime()));
+    }
+    else if (keyword.equalsIgnoreCase("privacy")) {
+      return String.format(botAnswer, getPrivacy(conversation.isPrivate()));
+    }
+    else if (keyword.equalsIgnoreCase("users")) {
+      int numberOfUsers = getNumberOfUsers(conversation);
+      if (numberOfUsers > 1) {
```

Comment:
nit: You could shorten these six lines to:
`return String.format(botAnswer, numberOfUsers) + (numberOfUsers > 1 ? "s." : ".");`
Or if you don't like the long line:
```
String suffix = (numberOfUsers > 1 ? "s." : ".");
return String.format(botAnswer, numberOfUsers) + suffix;
```